bonus/specs/bpd/winning_transactions_v2.json

```diff
@@ -0,0 +1,659 @@
+          "type": "number"
+        },
+        "trxDate": {
+          "format": "date-time",
+          "description": "timestamp dell'operazione di pagamento effettuata presso l'esercente",
+          "type": "string"
+        }
+      }
+    },
+    "TotalCashbackResource": {
+      "title": "TotalCashbackResource",
+      "required": [
+        "totalCashback",
+        "transactionNumber"
+      ],
+      "type": "object",
+      "properties": {
+        "totalCashback": {
+          "description": "Cashback transazione, indicato con segno '-' in caso di storno",
+          "type": "number"
+        },
+        "transactionNumber": {
+          "format": "int64",
+          "description": "Numero delle transazioni effettuate dall'utente",
+          "type": "integer"
+        }
+      },
+      "example": {
+        "totalCashback": 0.708740587556467,
+        "transactionNumber": 0
+      }
+    },
+    "WinningTransactionPageResource": {
+      "title": "WinningTransactionPage",
+      "required": [
+        "nextCursor",
+        "prevCursor",
+        "transactions"
+      ],
+      "type": "object",
+      "properties": {
+        "nextCursor": {
+          "format": "int32",
+          "description": "Id cursore da usare per la prossima richiesta per proseguire con la prossima pagina",
+          "type": "integer"
+        },
+        "prevCursor": {
+          "format": "int32",
+          "description": "Id cursore da usare per la richiesta precedente per proseguire con la pagina precedente",
+          "type": "integer"
+        },
+        "transactions": {
+          "type": "array"
+        }
+      },
+      "example": {
+        "nextCursor": 0,
+        "transactions": [
+          {
+            "count": 0,
+            "date": "string",
+            "transactions": [
+              {
+                "amount": 0,
+                "awardPeriodId": 0,
+                "cashback": 0,
+                "cashbackNorm": 0,
+                "circuitType": "string",
+                "hashPan": "string",
+                "idTrx": 0,
+                "idTrxAcquirer": "string",
+                "idTrxIssuer": "string",
+                "isPivot": false,
+                "trxDate": "string"
+              }
+            ]
+          }
+        ]
+      }
+    },
+    "WinningTransactionsOfTheDayResource": {
+      "title": "WinningTransactionsOfTheDay",
+      "description": "Oggetto che raggruppa le transazioni effettuate in una giornata",
+      "required": [
+        "date",
+        "transactions"
+      ],
+      "type": "object",
+      "properties": {
+        "date": {
+          "format": "date",
+          "description": "La data del giorno di riferimento",
+          "type": "string"
+        },
+        "transactions": {
+          "description": "Lista delle transazioni effettuate nella giornata di riferimento",
+          "type": "array"
+        }
+      },
+      "example": {
```

Comment:
This is the cause of the error

```
Error: Error: Error opening file ".../io-services-metadata/bonus/specs/bpd/string" 
ENOENT: no such file or directory, open '.../io-services-metadata/bonus/specs/bpd/string'
```
I don't know exactly when by I guess we can remove this entire block since it is an example